DiamondLightSource/SynchWeb
# Feature/lims 926/advanced filter do not show logic

**JIRA ticket**: [TICKET-926](https://jira.diamond.ac.uk/browse/LIMS-926)

**Summary**:

The advanced filters on the summary page do not show logic of selected filter parameters. 

**Changes**:
- Component added to component utils which takes an object and displays the values in a pill form in the DOM
- Component is an input where if clicked then will utilise the dialog-modal component 

**To test**:
- Go to summary page and click on Advanced Search
- Click on the + button and then on the input box where a dialogue box should appear
- Make a selection and, when clicking out of the component, check that the logic that appears is as expected.
